lrpar/src/lib/ctbuilder.rs

```diff
@@ -646,7 +641,12 @@ where
             }
         }
 
-        self.output_file(&grm, &stable, &derived_mod_name, outp, &cache)?;
+        self.output_file(&grm, &stable, &derived_mod_name, outp, &quote!{
+            // This declaration can be affected by the pretty printer.
+            // But we would hope the actual cache string is not.
+            #[allow(unused)]
+            const CACHE_INFO: &str = #cache;
```

Comment:
Looks like we can name this `const _: &str = #cache`.

I thought this wasn't a valid constant declaration, but it looks like it is now.

Reply:
Fixed by [3b860ca](https://github.com/softdevteam/grmtools/pull/525/commits/3b860ca64fb99da92f96dc8317eee3862bb3d119)